src/Functions/castOrDefault.cpp

```diff
@@ -376,66 +376,475 @@ SELECT accurateCastOrDefault('abc', 'UInt32')
+    FunctionDocumentation::Description toInt128OrDefault_description = R"(
+Like [`toInt128`](#toint128), this function converts an input value to a value of type [Int128](../data-types/int-uint.md) but returns the default value in case of an error.
+If no `default` value is passed then `0` is returned in case of an error.
+    )";
+    FunctionDocumentation::Syntax toInt128OrDefault_syntax = "toInt128OrDefault(expr[, default])";
+    FunctionDocumentation::Arguments toInt128OrDefault_arguments = {
+        {"expr", "Expression returning a number or a string representation of a number.", {"String", "Int", "Float"}},
+        {"default", "Optional. The default value to return if parsing is unsuccessful.", {"Int128"}}
+    };
+    FunctionDocumentation::ReturnedValue toInt128OrDefault_returned_value = {"Value of type Int128 if successful, otherwise returns the default value if passed or 0 if not.", {"Int128"}};
+    FunctionDocumentation::Examples toInt128OrDefault_examples = {
+        {"Successful conversion", "SELECT toInt128OrDefault('-128', CAST('-1', 'Int128'))", "-128"},
+        {"Failed conversion", "SELECT toInt128OrDefault('abc', CAST('-1', 'Int128'))", "-1"}
+    };
+    FunctionDocumentation::Category toInt128OrDefault_category = FunctionDocumentation::Category::TypeConversion;
+    FunctionDocumentation::IntroducedIn toInt128OrDefault_introduced_in = {21, 11};
+    FunctionDocumentation toInt128OrDefault_documentation = {toInt128OrDefault_description, toInt128OrDefault_syntax, toInt128OrDefault_arguments, toInt128OrDefault_returned_value, toInt128OrDefault_examples, toInt128OrDefault_introduced_in, toInt128OrDefault_category};
+
     factory.registerFunction("toInt128OrDefault", [](ContextPtr context)
-        { return std::make_shared<FunctionCastOrDefaultTyped>(context, "toInt128OrDefault", std::make_shared<DataTypeInt128>()); });
+        { return std::make_shared<FunctionCastOrDefaultTyped>(context, "toInt128OrDefault", std::make_shared<DataTypeInt128>()); },
+        toInt128OrDefault_documentation);
+    FunctionDocumentation::Description toInt256OrDefault_description = R"(
+Like [`toInt256`](#toint256), this function converts an input value to a value of type [Int256](../data-types/int-uint.md) but returns the default value in case of an error.
+If no `default` value is passed then `0` is returned in case of an error.
+    )";
+    FunctionDocumentation::Syntax toInt256OrDefault_syntax = "toInt256OrDefault(expr[, default])";
+    FunctionDocumentation::Arguments toInt256OrDefault_arguments = {
+        {"expr", "Expression returning a number or a string representation of a number.", {"String", "Int", "Float"}},
+        {"default", "Optional. The default value to return if parsing is unsuccessful.", {"Int256"}}
+    };
+    FunctionDocumentation::ReturnedValue toInt256OrDefault_returned_value = {"Value of type Int256 if successful, otherwise returns the default value if passed or 0 if not.", {"Int256"}};
```

Comment:
```suggestion
    FunctionDocumentation::ReturnedValue toInt256OrDefault_returned_value = {"Returns a value of type Int256 if successful, otherwise returns the default value if passed, or 0 if not.", {"Int256"}};
```